core/includes/image.inc

```diff
@@ -591,7 +591,8 @@ function image_add_svg_attributes($svg_content, array $attributes) {
         if (strlen($attributes['alt'])) {
           $title = $doc->createElement('title');
           $title->textContent = $attributes['alt'];
-          $doc->firstChild->prepend($title);
+          $first_child = $doc->childNodes->item(0)->childNodes->item(0);
+          $doc->childNodes->item(0)->insertBefore($title, $first_child);
```

Comment:
Thanks @indigoxela 🙏🏼 ...my solution for this in [backdrop/backdrop@`2c64fa8` (#4769)](https://github.com/backdrop/backdrop/pull/4769/commits/2c64fa8cf2ad139d09fb44e1830a0e5302618422) was slightly different, using `DOMNode::firstChild` (which is available since PHP5) instead of `DOMNode::childNodes->item(0)`. That simplifies the above to a single line:
```suggestion
          // Since DOMDocument::prepend() is not available in PHP versions prior
          // to v8, we are using this trickery with DOMNode::insertBefore() and
          // DOMNode::firstChild.
          $doc->firstChild->insertBefore($title, $doc->firstChild->firstChild);
```

Do you think better to use that instead? (plus the inline comment about php8, so that people don't attempt to change that to `DOMDocument::prepend()` in the future).